lib/sawyer/agent.rb

```diff
@@ -1,4 +1,4 @@
-require 'faraday'
+require 'hurley'
 require 'addressable/template'
```

Comment:
Should we require addressable-uri before hurley?  Since we're using Addressable anyway, might as well use it for URI parsing too.